### DISCUSSION TITLE
Oxalis - Tomcat 10 support

### DISCUSSION THREAD

**Discussion by kirti2001** [OxalisCommunity/oxalis#677] [General]
Hi,

Do you have tomcat10 on roadmap for Oxalis? Phoss SMP version 7 or greater than 7 supports tomcat 10 and Phoss SMP 7.1.0 replaced SHA-1 with SHA-256 for the signature which is one of the peppol compliances.

Thanks,
Kirti

**Comment by aaron-kumar** [comment]
Please refer refer June 2024 Oxalis Technical Meeting Notes
